sphinx/directives/__init__.py

```diff
@@ -55,7 +55,7 @@ class ObjectDescription(SphinxDirective, Generic[ObjDescT]):
     required_arguments = 1
     optional_arguments = 0
     final_argument_whitespace = True
-    option_spec: OptionSpec = {
+    option_spec: ClassVar[OptionSpec] = {
```

Comment:
would it maybe be best to just change to:

```python
OptionSpec = ClassVar[dict[str, Callable[[str], Any]]]
```

this is the only purpose we use this type annotation for, so it would reduce boiler-plate

Reply:
I personally prefer not. I feel that using `ClassVar` should be used ... in a class and not outside as an alias (an alias that is also in another module). In addition, you may use this in other places (e.g., as the type of argument of some `__init_subclass__` for instance).